yt_dlp/extractor/npo.py

```diff
@@ -179,15 +199,13 @@ def suitable(cls, url):
 
     def _real_extract(self, url):
         video_id = self._match_id(url)
-        return self._get_info(url, video_id) or self._get_old_info(video_id)
 
-    def _get_info(self, url, video_id):
-        token = self._download_json(
+        self._download_json(
             'https://www.npostart.nl/api/token', video_id,
```

Comment:
```suggestion
        self._request_webpage(
            'https://www.npostart.nl/api/token', video_id,
```
There is no read to parse or even read the result if it's not gonna be used